src/nvim/ex_docmd.c

```diff
@@ -5640,6 +5641,20 @@ static void ex_syncbind(exarg_T *eap)
   }
 }
 
+void do_read_cmd(exarg_T *eap)
+{
+  char *cmd = eap->arg + 1;
+  StringBuilder put_cmd = KV_INITIAL_VALUE;
+  kv_printf(put_cmd, "try | ");
+  kv_printf(put_cmd, "%dput=execute('%s') | ", eap->line2, cmd);
+  kv_printf(put_cmd, "execute 'norm! `.' | ");
```

Comment:
`execute()` seems to add an extra empty line above the output, so I have to jump back to the top of the inserted text and delete this line, so that we get behaviour that is more similar to reading a file.